bundles/binding/org.openhab.binding.caldav-personal/src/main/java/org/openhab/binding/caldav_personal/internal/CalDavConfig.java

```diff
@@ -76,77 +93,82 @@ public Value getValue() {
-        result = prime * result
-                + ((filterName == null) ? 0 : filterName.hashCode());
+        result = prime * result + ((filterCategory == null) ? 0 : filterCategory.hashCode());
+        result = prime * result + ((filterName == null) ? 0 : filterName.hashCode());
         result = prime * result + ((type == null) ? 0 : type.hashCode());
         result = prime * result + ((value == null) ? 0 : value.hashCode());
         return result;
     }
 
     @Override
     public boolean equals(Object obj) {
-        if (this == obj)
+        if (this == obj) {
             return true;
-        if (obj == null)
+        }
+        if (obj == null) {
             return false;
-        if (getClass() != obj.getClass())
+        }
+        if (getClass() != obj.getClass()) {
             return false;
+        }
         CalDavConfig other = (CalDavConfig) obj;
         if (calendar == null) {
-            if (other.calendar != null)
+            if (other.calendar != null) {
                 return false;
-        } else if (!calendar.equals(other.calendar))
+            }
+        } else if (!calendar.equals(other.calendar)) {
             return false;
-        if (eventNr != other.eventNr)
+        }
+        if (eventNr != other.eventNr) {
             return false;
+        }
         if (filterCategory == null) {
-            if (other.filterCategory != null)
+            if (other.filterCategory != null) {
                 return false;
-        } else if (!filterCategory.equals(other.filterCategory))
+            }
+        } else if (!filterCategory.equals(other.filterCategory)) {
             return false;
+        }
         if (filterName == null) {
-            if (other.filterName != null)
+            if (other.filterName != null) {
                 return false;
-        } else if (!filterName.equals(other.filterName))
+            }
+        } else if (!filterName.equals(other.filterName)) {
             return false;
-        if (type != other.type)
+        }
+        if (type != other.type) {
             return false;
-        if (value != other.value)
+        }
+        if (value != other.value) {
             return false;
+        }
         return true;
     }
 
     public int getUniqueEventListKey() {
         final int prime = 31;
         int result = 1;
-        result = prime * result
-                + ((calendar == null) ? 0 : calendar.hashCode());
-        result = prime * result
-                + ((filterCategory == null) ? 0 : filterCategory.hashCode());
-        result = prime * result
-                + ((filterName == null) ? 0 : filterName.hashCode());
+        result = prime * result + ((calendar == null) ? 0 : calendar.hashCode());
+        result = prime * result + ((filterCategory == null) ? 0 : filterCategory.hashCode());
+        result = prime * result + ((filterName == null) ? 0 : filterName.hashCode());
         return result;
     }
-    
-    
+
 }
```

Comment:
Adding the new member `categoriesFiltersAny` to this class should result in changes to `hashCode`, `equals`, `toString` and possibly the `getUniqueEventListKey` methods.